mllib/src/main/scala/org/apache/spark/ml/feature/StandardScaler.scala

```diff
@@ -29,7 +29,22 @@ import org.apache.spark.sql.types.{StructField, StructType}
 /**
  * Params for [[StandardScaler]] and [[StandardScalerModel]].
  */
-private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol
+private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol {
+  
+  /**
+   * Whether to center the data before scaling
```

Comment:
Could you please state the default here?
Also, can you please add this warning (copied & edited from the class doc in the old class):
```
When `withMean` is true, the output vectors will be dense, so this does not work on sparse input and will raise an exception.  When `withMean` is false, the input can be dense or sparse.
```